metacat-main/src/main/java/com/netflix/metacat/main/configs/ServicesConfig.java

```diff
@@ -79,13 +83,50 @@ public UserMetadataService userMetadataService(
     }
 
     /**
+     * Tag service.
+     *
+     * @param dataSourceManager The datasource manager to use
+     * @param config            System config to use
+     * @param metacatJson       Json Utilities to use
+     * @return User metadata service based on MySql
+     */
+    @Bean
+    @ConditionalOnMissingBean
```

Comment:
Probably need to say `type = TagService.class` in this annotation otherwise I think it just looks for a missing bean of name `tagService`